src/rpc/RPCHelpers.cpp

```diff
@@ -1,7 +1,7 @@
 //------------------------------------------------------------------------------
 /*
     This file is part of clio: https://github.com/XRPLF/clio
-    Copyright (c) 2022, the clio developers.
+    Copyright (c) 2023, the clio developers.
```

Comment:
The year here is the year the file appeared, so you don't need this change.

Reply:
done